src/components/search/SearchBox.tsx

```diff
@@ -15,15 +15,20 @@ type SearchBoxProps = {
   title: string;
   subTitle: string;
   placeholderText?: string;
+  initialParamsLoaded?: boolean;
 };
 
 const SearchBox = (props: SearchBoxProps) => {
-  const { title, subTitle, placeholderText } = props;
+  const { title, subTitle, placeholderText, initialParamsLoaded } = props;
 
   const searchActions = useSearchActions();
 
   // When the FilterSearch component updates the search state with the users selection execute a new search.
   useEffect(() => {
+    if (!initialParamsLoaded) {
```

Comment:
should this be a dep of the useEffect hook so it reruns when it switches from false to true?